Mathlib/Tactic/NormNum/Basic.lean

```diff
@@ -927,3 +927,22 @@ such that `norm_num` successfully recognises both `a` and `b`. -/
   have nc : Q(ℕ) := mkRawNatLit (na.natLit! % nb.natLit!)
   let r : Q(Nat.mod $na $nb = $nc) := (q(Eq.refl $nc) : Expr)
   return (.isNat sℕ nc q(isNat_natMod $pa $pb $r) : Result q($a % $b))
+
+theorem isNat_natDiv : {a b : ℕ} → {a' b' c : ℕ} →
+    IsNat a a' → IsNat b b' → Nat.div a' b' = c → IsNat (a / b) c
+  | _, _, _, _, _, ⟨rfl⟩, ⟨rfl⟩, rfl => ⟨by aesop⟩
+
+/-- The `norm_num` extension which identifies expressions of the form `Nat.div a b`,
+such that `norm_num` successfully recognises both `a` and `b`. -/
+@[norm_num (_ : ℕ) % _, HDiv.hDiv (_ : ℕ) _, Nat.div _ _] def evalNatDiv :
```

Comment:
I think the first two patterns here are the same. You probably want `Div.div` for the second pattern

Reply:
oops, looks like the first pattern was wrong even